packages/daimo-common/src/op.ts

```diff
@@ -82,6 +84,11 @@ interface OpEventBase {
   blockNumber?: number;
   blockHash?: string;
   logIndex?: number;
+
+  /* Fees paid to paymaster for corresponding userop, in USDC amount.
+   * Estimate if pending, actual if not. 0 if paymaster wasn't used.
+   */
+  feeAmount?: number;
```

Comment:
nit but i'd call this `paymasterFeeAmount`, otherwise would be surprising if a userop with fees had feeAmount 0

Reply:
this would take a migration for the existing transfer ops saved, so not going to touch it in this PR, but good point for future cleanup.